vividus-tests/src/main/resources/story/integration/TableTransformers.story

```diff
@@ -321,3 +321,29 @@ Examples:
 |2       |
 |1       |
 |0       |
+
+Scenario: Verify INNER_JOIN transformer with empty table (should not be executed)
+Meta:
+    @requirementId 1810
+Then `should` is equal to `be skipped`
+Examples:
+{transformer=INNER_JOIN, leftTableJoinColumn=joinID, rightTableJoinColumn=joinID1, tables=/data/for-inner-join-transformer.table}
+|joinID1 |
+
+Scenario: Verify INNER_JOIN transformer with table body
+Meta:
+    @requirementId 1810
+When I initialize scenario variable `innerJoinTable` with values:
+{transformer=INNER_JOIN, leftTableJoinColumn=joinID, rightTableJoinColumn=joinID, tables=/data/for-inner-join-transformer.table}
```

Comment:
if join column keys are different, will both of these keys present in resulting table?

Reply:
yes, both of them will be present